testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithDeploymentConfigTest.java

```diff
@@ -567,6 +618,17 @@ public void testCorsRequestWithEnableCorsWithInvalidOrigin() throws Exception {
         }
     }
 
+    @Test
+    @InSequence(27)
+    public void testFormWithOidc() throws Exception {
```

Comment:
Just as a sanity check, were you able to verify that this test fails without the changes in `UndertowDeploymentProcessor` and passes after these changes?

Reply:
tested.  It failed as it should.